app/views/dashboard/schools/class_rooms/index.html.slim

```diff
@@ -23,10 +23,12 @@
                         .row.p-2.d-flex data-test="student-not-in-class-room-#{student.id}"
                           .col-7.d-flex.align-items-center.font-weight-bold.student-name= student.name
                           .col-4
+                            / autocomplete off from https://stackoverflow.com/questions/4831848/firefox-ignores-option-selected-selected
```

Comment:
Je suis allé voir par curiosité, et je me demandais si l'autre réponse sur Stack Overflow fonctionnait : 

_In firefox, I've notice that the "selected" attribute will not work unless you place the select inside a form, where the form has a name attribute._